### PR TITLE
Add JAX backend support to OperatorPredictor

### PR DESCRIPTION
**Pull request by bonneted**

### CODE REVIEW

File: deepxde/callbacks.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from . import config
 from . import gradients as grad
 from . import utils
-from .backend import backend_name, tf, torch, paddle
+from .backend import backend_name, tf, torch, paddle, jax
```
Comment (lululxvi) [inline deepxde/callbacks.py:9]:
```
from .backend import backend_name, jax, paddle, tf, torch
```

File: deepxde/callbacks.py [hunk 2]
```diff
@@ -427,6 +435,8 @@ def on_predict_end(self):
             self.model.net.eval()
             outputs = self.model.net(self.x)
             self.value = utils.to_numpy(self.op(self.x, outputs))
+        elif backend_name == "jax":
```
Comment (lululxvi) [inline deepxde/callbacks.py:438]: Move jax before paddle.

**Comment by bonneted** [comment]
Do you want me to remove also the TODO, or keep it for MXNet ?

**Comment by lululxvi** [comment]
> Do you want me to remove also the TODO, or keep it for MXNet ?

Remove the TODO.

**Pull request merged**